changelog/4913.bugfix.rst

```diff
@@ -0,0 +1 @@
+Fix pytest tests invocation with custom ``PYTHONPATH``.
```

Comment:
The `bugfix` type is ~usually for externally facing bugs -- I'd use the `trivial` changelog type here since this is internal to pytest's testsuite